client/ladder/ladder.tsx

```diff
@@ -620,18 +625,39 @@ export function LadderTable(props: LadderTableProps) {
         />
         <DivisionSelect
           dense={true}
-          label={'Division'}
+          label={t('ladder.divisionLabel', 'Division')}
           value={filteredDivision}
           onChange={onFilteredDivisionChange}
           allowErrors={false}>
-          <SelectOption value={DivisionFilter.All} text={'All'} />
-          <SelectOption value={DivisionFilter.Champion} text={'Champion'} />
-          <SelectOption value={DivisionFilter.Diamond} text={'Diamond'} />
-          <SelectOption value={DivisionFilter.Platinum} text={'Platinum'} />
-          <SelectOption value={DivisionFilter.Gold} text={'Gold'} />
-          <SelectOption value={DivisionFilter.Silver} text={'Silver'} />
-          <SelectOption value={DivisionFilter.Bronze} text={'Bronze'} />
-          <SelectOption value={DivisionFilter.Unrated} text={'Unrated'} />
+          <SelectOption value={DivisionFilter.All} text={t('ladder.divisionOption.all', 'All')} />
+          <SelectOption
+            value={DivisionFilter.Champion}
+            text={t('ladder.divisionOption.champion', 'Champion')}
```

Comment:
These should use text that is shared among all things that need to refer to these divisions, this is not specific to this dropdown